inc/Classes/Display.php

```diff
@@ -1292,7 +1292,7 @@ public function FetchUserIcon($userid, $username = '')
             $username = '<i>System</i>';
         }
 
-        $smarty->assign('userid', $userid);
+        $smarty->assign('userid', htmlspecialchars($userid));
```

Comment:
Isn't this always an integer?

Reply:
This change was copied over by hand and I added it into the wrong line.
`$userid` is an integer, `$username` is (where it belongs) is a string with potentially html characters.
Will be changed.